packages/camera/camera_avfoundation/ios/Classes/FLTCamMediaSettings.m

```diff
@@ -0,0 +1,32 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import "FLTCamMediaSettings.h"
+
+@implementation FLTCamMediaSettings
+
+#define AssertPositiveNumberOrNil(param)                                                        \
```

Comment:
Does this need to be a macro?  Can it be an inlined function?

Reply:
This is for `@ #param` token stringification. Should i refactor this to  something like:

```
NS_INLINE void AssertPositiveNumberOrNil(id param, const char* paramName) {
  if (param != nil) {
    NSAssert([param isKindOfClass:[NSNumber class]], @"%s is not a number: %@", paramName, param);
    ...
```
?

Reply:
refactored as inlined function.